airbyte-integrations/connectors/source-zendesk-talk/source_zendesk_talk/manifest.yaml

```diff
@@ -0,0 +1,1404 @@
+type: DeclarativeSource
+
+definitions:
+  authenticator:
+    class_name: source_zendesk_talk.components.ZendeskTalkAuthenticator
```

Comment:
I did of course look at that component, but I chose not to use it because I did not see a way to define both the legacy and updated config structure used with `BasicHttpAuthenticator` the way the python connector did.

Reply:
> if we need to revert, we can't. With such a big change as changing the implementation almost completely, it seems rather risky

This was my thinking as well when I opted to dump the existing nonstandard authentication logic into a custom component instead of just keeping the two non-legacy auth options, which _would_ be possible to do in the manifest with `SelectiveAuthenticator`—though given the sparse documentation and lack of existing usages in the repo to use as reference, it might also be a bit slower to build it that way.